bibliopixel/__main__.py

```diff
@@ -0,0 +1,43 @@
+import platform
+
+MINIMUM_VERSION = '3.4.0'
+
+if platform.python_version() < MINIMUM_VERSION:
+    raise EnvironmentError(
+        '%s needs Python %s or later: your Python version is %s' % (
+            __package__, MINIMUM_VERSION, platform.python_version()))
+
+try:
+    import argparse
+    import argcomplete
+    import sys
+    from bibliopixel import __description__, __url__
```

Comment:
How can these imports possibly fail?  

Also, we do not use the "one import per line" convention in the project as it is today.